config/environments/development.rb

```diff
@@ -38,4 +39,8 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.x.saml_issuer = 'urn:gov:gsa:SAML:2.0.profiles:sp:sso:localhost-rails'
+  config.x.idp_url = 'http://localhost:3000/api/saml/auth'
+  config.x.idp_cert_fingerprint = 'E6:12:DC:4D:93:66:02:E0:74:AE:B8:2C:F8:2C:3A:C5:AA:20:3D:60'
 end
```

Comment:
I would put these settings in a YAML file, such as `secrets.yml` that Rails provides. That way, you only have to look at one file to make changes in all environments. `secrets.yml` allows you to specify different settings for test, dev, and production all in the same file.